packages/strapi-generate-new/lib/before.js

```diff
@@ -45,8 +48,246 @@ module.exports = (scope, cb) => {
+          settings: {
+            client: answers.client.database
+          },
+          options: {}
+        });
+
+        resolve();
+      });
+    });
+
+    formSteps = formSteps.then(() => {
+      const asyncFn = [
+        new Promise(resolve => {
+          inquirer
+          .prompt([
+            {
+              type: 'input',
+              prefix: '',
+              name: 'name',
+              message: 'Database name:',
+              default: _.get(scope.database, 'database', 'strapi')
+            },
+            {
+              type: 'input',
+              prefix: '',
+              name: 'host',
+              message: 'Host:',
+              default: _.get(scope.database, 'host', 'localhost')
+            },
+            {
+              type: 'input',
+              prefix: '',
+              name: 'port',
+              message: 'Port:',
+              default: (answers) => {
+                if (_.get(scope.database, 'port')) {
+                  return scope.database.port;
+                }
+
+                const ports = {
+                  mongo: 27017,
+                  postgres: 5432,
+                  mysql: 3306,
+                  sqlite3: 1433,
+                  redis: 6379
+                };
+
+                return ports[scope.client.database];
+              }
+            },
+            {
+              type: 'input',
+              prefix: '',
+              name: 'username',
+              message: 'Username:',
+              default: _.get(scope.database, 'username', undefined)
+            },
+            {
+              type: 'input',
+              prefix: '',
+              name: 'password',
+              message: 'Password:',
+              default: _.get(scope.database, 'password', undefined)
+            }
+          ])
+          .then(answers => {
+            scope.database.host = answers.host;
+            scope.database.port = answers.port;
+            scope.database.database = answers.name;
+            scope.database.username = answers.username;
+            scope.database.password = answers.password;
+
+            resolve();
+          });
+        }),
+        new Promise(resolve => {
+          let cmd = `npm install --prefix ${scope.rootPath}_ ${scope.client.connector}@alpha`;
+          if (scope.client.module) {
+            cmd += ` ${scope.client.module}`;
+          }
+
+          exec(cmd, () => {
+            if (scope.client.module) {
+              const lock = require(`${scope.rootPath}_/node_modules/${scope.client.module}/package.json`);
+              scope.client.version = lock.version;
+            }
+
+            resolve();
+          });
+        })
+      ];
+
+      Promise.all(asyncFn)
+      .then(() => {
+        if (scope.client.connector === 'strapi-bookshelf') {
+          const knex  = require(`${scope.rootPath}_/node_modules/knex`)({
+            client: scope.client.module,
+            connection: scope.database
```

Comment:
At this point a new project with a remote postgres database will fail because connection.ssl is false.

I added this code in before.js to able to generate a new project with postgres : 

```
scope.database.settings.ssl = scope.database.settings.client === 'postgres' ? true : false;
```